### PR TITLE
CLOUDP-253857: [Docs Feedback] atlas events projects list

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ CLOUDP-253857

Docs feedback

### CODE REVIEW

**Review by tibulca — APPROVED** [review]
LGTM

File: internal/cli/events/orgs_list.go [hunk 1]
```diff
@@ -105,7 +105,10 @@ func OrgListBuilder() *cobra.Command {
 		Aliases: []string{"ls"},
 		Args:    require.NoArgs,
 		Example: `  # Return a JSON-formatted list of events for the organization with the ID 5dd5a6b6f10fab1d71a58495:
-  atlas events organizations list --orgId 5dd5a6b6f10fab1d71a58495 --output json`,
+  atlas events organizations list --orgId 5dd5a6b6f10fab1d71a58495 --output json
+  
+  # Return a JSON-formatted list of events between 2024-03-18T14:40:03-0000 and 2024-03-18T15:00:03-0000 and for the project with the ID 5e2211c17a3e5a48f5497de3 bew
```
Comment (jwilliams-mongo) [inline internal/cli/events/orgs_list.go:110]: is this part of the ID?
```suggestion
  # Return a JSON-formatted list of events between 2024-03-18T14:40:03-0000 and 2024-03-18T15:00:03-0000 and for the project with the ID 5e2211c17a3e5a48f5497de3
```

File: internal/cli/events/projects_list.go [hunk 2]
```diff
@@ -105,7 +105,10 @@ func ProjectListBuilder() *cobra.Command {
 		Aliases: []string{"ls"},
 		Args:    require.NoArgs,
 		Example: `  # Return a JSON-formatted list of events for the project with the ID 5e2211c17a3e5a48f5497de3:
-  atlas events projects list --Id 5e2211c17a3e5a48f5497de3 --output json`,
+  atlas events projects list --Id 5e2211c17a3e5a48f5497de3 --output json
+
+  # Return a JSON-formatted list of events between 2024-03-18T14:40:03-0000 and 2024-03-18T15:00:03-0000 and for the project with the ID 5e2211c17a3e5a48f5497de3 bew
```
Comment (jwilliams-mongo) [inline internal/cli/events/projects_list.go:110]:
```suggestion
  # Return a JSON-formatted list of events between 2024-03-18T14:40:03-0000 and 2024-03-18T15:00:03-0000 and for the project with the ID 5e2211c17a3e5a48f5497de3
```

**Review by jwilliams-mongo — DISMISSED** [review]
two minor comments then LGTM

File: internal/cli/events/projects_list.go (hunk 2 above)
Comment (gssbzn) [reply internal/cli/events/projects_list.go:110]: good eye

**Review by lovisaberggren — APPROVED** [review]

**Pull request merged**